app/assets/styles/themes/northernireland/theme.scss

```diff
@@ -1,2 +1,16 @@
 @import "../../themes-common/config";
 @import "../../themes-common/ui";
+
+/**
+ * Theme override
+ */
+
+.logo__img {
+  width: 280px;
+}
+
+.footer_logo {
+  width: 190px;
+  height: auto;
+  margin-left: 1rem;
+}
```

Comment:
Move this to ` app/assets/styles/themes/northernireland/components/_footer.scss` then imported here with `@import "components/footer"`